website/docs/release_notes.md

```diff
@@ -8,6 +8,47 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v1.2.1](https://github.com/VirtusLab/scala-cli/releases/tag/v1.2.1)
+
+### Support for Scala.js 1.16.0
```

Comment:
Don't we also start supporting 0.5.0 via Bloop upgrade? Maybe worth mentioning, probably also worth a test

Reply:
0.5.0? of what?